src/windowsMain/kotlin/file.kt

```diff
@@ -93,6 +94,68 @@ actual class File actual constructor(private val rawPath: String) {
     override fun toString(): String {
         return path
     }
+
+    @OptIn(ExperimentalForeignApi::class)
+    actual fun move(dest: File): Boolean {
+        memScoped {
+            val pathW = path.wcstr.ptr
+            val destW = dest.path.wcstr.ptr
+            val flags = MOVEFILE_REPLACE_EXISTING.toUInt()
+            return MoveFileEx!!(pathW, destW, flags) != 0
```

Comment:
Should we change the other Windows function calls currently using `?.let` to instead use `!!`? We probably want the program to fail fast (crash) if these Windows functions aren't actually available, no?